src/main/java/org/folio/edge/core/Handler.java

```diff
@@ -180,6 +180,6 @@ protected void internalServerError(RoutingContext ctx, String msg) {
 
   @FunctionalInterface
   public interface TwoParamVoidFunction<A, B> {
-    public void apply(A a, B b);
+    void apply(A a, B b);
   }
```

Comment:
Can TwoParamVoidFunction been replaced by java.util.function.BiConsumer?

Reply:
It'd be a breaking change, right?  So if we're going to do this let's make it part of v4

Reply:
Public is redundant here. Not a breaking change